lib/src/notification/notification.component.spec.ts

```diff
@@ -0,0 +1,251 @@
+import { Component } from "@angular/core";
+import { async, ComponentFixture, TestBed, fakeAsync, tick } from "@angular/core/testing";
+import { By } from "@angular/platform-browser";
+import {
+    NotificationComponent,
+    NotificationAction,
+    NotificationPosition,
+    NotificationTheme,
+    NotificationStyle,
+} from "./notification.component";
+import { CommonModule } from "@angular/common";
+
+@Component({
+    selector: "test-sebng-notification",
+    template: `
+        <sebng-notification
+            [className]="className"
+            [dismissable]="dismissable"
+            [dismissTimeout]="dismissTimeout"
+            [message]="message"
+            (onClick)="onClick()"
+            (onDismiss)="onDismiss($event)"
+            [persist]="persist"
+            [position]="position"
+            [style]="style"
+            [theme]="theme"
+            [title]="title"
+            [toggle]="toggle"
+            [actions]="actions"
+        >
+            <div class="testing">test</div></sebng-notification
+        >
+    `,
+})
+class NotificationTestComponent {
+    className?: string;
+    dismissable?: boolean;
+    dismissTimeout?: number;
+    message?: string;
+    onClick() {}
+
+    onDismiss() {}
+    persist?: boolean;
+    position?: NotificationPosition;
+    style?: NotificationStyle;
+    theme?: NotificationTheme;
+    title?: string;
+    toggle: boolean;
+    actions?: Array<NotificationAction>;
+}
+
+describe("NotificationComponent", () => {
+    let component: NotificationTestComponent;
+    let fixture: ComponentFixture<NotificationTestComponent>;
+    let onDismiss: jasmine.Spy;
+    let onClick: jasmine.Spy;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            imports: [CommonModule],
+            declarations: [NotificationComponent, NotificationTestComponent],
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(NotificationTestComponent);
+        component = fixture.componentInstance;
+
+        component.toggle = true;
+        onDismiss = spyOn(component, "onDismiss");
+        onClick = spyOn(component, "onClick");
+        component.message = "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.";
+        component.actions = [
+            { text: "Yes, I'm in", action: () => this.onToggle2(false) },
```

Comment:
where is onToggle2 declared, we should not need 2 at the end of the function name